### PR TITLE
Add instructions for Google Cloud DNS forwarding

### PR DESCRIPTION
**Pull request by marciocloudflare**
This replicates PR-1992 but with the new 1.1.1.1 docs' structure.

### CODE REVIEW

**Review by elithrar — APPROVED** [review]

File: products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+order:
+pcx-content: how-to
+---
+
+# Google Cloud
+
+Google Cloud supports configuring [outbound server policy](https://cloud.google.com/dns/docs/server-policies-overview#dns-server-policy-out) within Cloud DNS. Policies are applied per Virtual Private Cloud (VPC) network, and will apply to all resources within that VPC network, including any existing virtual machines.
```
Comment (kodster28) [inline products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]:
```suggestion
Google Cloud supports configuring [outbound server policy](https://cloud.google.com/dns/docs/server-policies-overview#dns-server-policy-out) within Cloud DNS. Policies are applied per Virtual Private Cloud (VPC) network, and will affect all resources within that VPC network, including any existing virtual machines.
```

Comment (kodster28) [reply products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]: Not sure if that's the right word, but trying to get away from the same verb 2x in a sentence

File: products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+<Aside type="note">
+
+If you're using [Cloudflare for Teams](https://developers.cloudflare.com/cloudflare-one/), you can use the assigned [locations](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/locations) to apply custom [DNS policies](https://developers.cloudflare.com/cloudflare-one/policies/filtering/dns-policies-builder) via Gateway.
```
Comment (kodster28) [inline products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]:
```suggestion
If you are using [Cloudflare for Teams](https://developers.cloudflare.com/cloudflare-one/), you can choose assigned [locations](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/locations) to apply custom [DNS policies](https://developers.cloudflare.com/cloudflare-one/policies/filtering/dns-policies-builder) via Gateway.
```

Comment (kodster28) [reply products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]: Same as above

File: products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+</Aside>
+
+To configure 1.1.1.1 for your Google Cloud VPC network(s):
+
+1. Open the [Google Cloud Console](https://console.cloud.google.com).
+
+1. Head to **Network Services** > **Cloud DNS** and click on [**DNS Server Policies**](https://console.cloud.google.com/net-services/dns/policies).
```
Comment (kodster28) [inline products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]:
```suggestion
1. Navigate to **Network Services** > **Cloud DNS** and click [**DNS Server Policies**](https://console.cloud.google.com/net-services/dns/policies).
```

File: products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Click **Create Policy**.
+
+1. Provide a name for your Policy (such as `cloudflare-1-1-1-1`), and select the VPC network or networks that this policy applies to.
```
Comment (kodster28) [inline products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md]:
```suggestion
1. Provide a name for your Policy (such as `cloudflare-1-1-1-1`) and select associated VPC network or networks.
```

**Review by kodster28 — APPROVED** [review]

**Pull request merged**